### DISCUSSION TITLE
Question: Screenshot tool with sharing feature

### DISCUSSION THREAD

**Discussion by rubiin** [prasanthrangan/hyprdots#1369] [Q&A] [answered]
Wondering if there is a tool out there for hyprland that allows sharing the screenshot. While I was using kde, I used spectacle and it had a feature to share the taken screenshot to `imgur`

**Comment by kRHYME7** [comment]
Maybe you can find something here 
https://github.com/hyprland-community/awesome-hyprland

Can we script this functionality?

**Reply by kRHYME7** [reply]
yeah we can upload it https://askubuntu.com/questions/417295/how-to-take-a-screenshot-and-then-upload-the-image-to-imgur-through-terminal

**Reply by rubiin** [reply]
yeah we can, imgur provides api which can be accesses via curl. It even has documentation
https://apidocs.imgur.com

**Reply by kRHYME7** [reply, marked as answer]
Yey Hmm we can add a flag for screenshot something ` --share [any.api.com] as long as they work they same.
